app/router.js

```diff
@@ -8,6 +8,7 @@ const Router = Ember.Router.extend({
 
 Router.map(function() {
   this.route('reminders', { path: '/' })
+  this.route('reminder', {path: 'reminders/:reminder_id'});
```

Comment:
Close! Nest the individual `reminder` route within the parent `reminders` route. Check out an example [here](https://guides.emberjs.com/v2.11.0/routing/defining-your-routes/#toc_nested-routes). On the same vein, if you use the CLI command `ember g route reminders/reminder` it will do the heavy lifting for you.